### ISSUE TITLE
Sub rogue-failed to make a recommendation-hekili

### ISSUE THREAD

**Issue by UmutTemizyurek** [Hekili/hekili#2530]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

For my subtlety rogue, while I am playing and following the hekili recommendations, the icons of my abilities start disappearing. Sometimes it only happens to one of them while other times the whole icons disappears for couple seconds.

### How to Reproduce

1. Get into a subtlety rogue
2. go to a practice dummy
3. follow recommendation

### Snapshot (Link)

https://pastebin.com/YgcgspPe

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/1H4fGcyJhCcubs8mxsSzGj

### Contact Information

Destr0#9932 (discord)

**Comment by Hekili** [comment]
Thanks.

```
warnings: [#1] Unable to compile '4))' - [string "return 4))"]:1: '<eof>' expected near ')' (loadstring-n)
From: 4)) - 25+safenum(talent.vigor.enabled)*20+safenum(talent.master_of_shadows.enabled)*20+safenum(talent.shadow_focus.enabled)*25+safenum(talent.alacrity.enabled)*20+25*safenum((spell_targets.shuriken_storm + 0.01
[#2] Unable to compile '((spell_targets.shuriken_storm+0.01' - [string "return ((spell_targets.shuriken_storm+0.01"]:1: ')' expected near '<eof>' (loadstring-n)
From: 4)) - 25+safenum(talent.vigor.enabled)*20+safenum(talent.master_of_shadows.enabled)*20+safenum(talent.shadow_focus.enabled)*25+safenum(talent.alacrity.enabled)*20+25*safenum((spell_targets.shuriken_storm + 0.01
[#3] Variable recheck function for 25 + talent.vigor.enabled * 20 + talent.master_of_shadows.enabled * 20 + talent.shadow_focus.enabled * 25 + talent.alacrity.enabled * 20 + 25 * (spell_targets.shuriken_storm >= 4) ( 4))-25+safenum(talent.vigor.enabled)*20+safenum(talent.master_of_shadows.enabled)*20+safenum(talent.shadow_focus.enabled)*25+safenum(talent.alacrity.enabled)*20+25*safenum((spell_targets.shuriken_storm+0.01) was unsuccessful somehow.
```

This means my recent update that tries to figure out the timing for condition changes is broken.  I'll look into it.

**Comment by Hekili** [comment]
Duplicate of #2527

**Comment by Hekili** [comment]
Resolved in v10.0.7-1.0.10.
